vng_api_common/schema.py

```diff
@@ -358,3 +358,18 @@ def get_error_codes(self):
 
         status_codes = sorted({e.status_code for e in exception_klasses})
         return status_codes
+
+    def _get_serializer_field_meta(self, field, direction):
+        meta = super()._get_serializer_field_meta(field, direction)
+        title = meta.get("title", "")
+        if not title:
+            if field.label:
+                meta["title"] = field.label
+            else:
+                try:
+                    meta["title"] = getattr(
+                        field.parent.Meta.model, str(field.source)
+                    ).field.verbose_name
+                except (AttributeError, KeyError):
+                    meta["title"] = field.field_name
```

Comment:
Ik zou proberen om dit iets wat meer leesbaar te maken:
```suggestion
                source = getattr(field.parent.Meta.model, str(field.source), None)
                if source and hasattr(source.field, "verbose_name"):
                    if source.field.verbose_name:
                        meta["title"] = source.field.verbose_name
```

Reply:
zonder try/except word deze wel lastig want niet elke parent heeft een Meta en niet elke source heeft een field. Hierdoor krijg je uiteindelijk well erg veel if stateaments. Of heb je dat liever dan een enkele try/except?

Reply:
Ik denk dat dat alsnog wat leesbaarder wordt dan dat het nu is. De redenering die je geeft kan ik niet zo een twee drie opmaken uit de `try`/`catch` omdat het allemaal afgevangen word.